### PR TITLE
Added to MODULES.md

### PR DESCRIPTION
**Pull request by makeworld-the-better-one**
I've added a Table of Contents, SSB peering info, and links to the public peers repos for both CJDNS and Yggdrasil.

The TOC is generated with [this](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) VSCode add on, I would recommend it for markdown in general.

### CODE REVIEW

File: docs/MODULES.md [hunk 1]
```diff
@@ -186,10 +211,12 @@ If you use this rule, there is no point in having any other Yggdrasil client rul
...
 You can specify a protocol, but that would limit the ports that are open.
 
-## Adding deprecated.systems peer info into CJDNS and Yggdrasil (Optional)
+## CJDNS and Yggdrasil public peering (Optional)
 
 #### CJDNS
 
+Other peers can be found in the [this](https://github.com/hyperboria/peers) repo of peers. Try to connect to only a few peers, and ones that are close to where you live. One peer is already shown below for you.
+
 Go to [Deprecated Systems](https://deprecated.systems/) website. You will see the following information:
```
Comment (darkdrgn2k) [inline docs/MODULES.md]: I think the docs should be more generic and not be linked to tomesh resources. 

Specifically listing deprecated.systems here may be inappropriate. I dont think we want these resources flooded with random people by advertising it so publicly.

Comment (darkdrgn2k) [reply docs/MODULES.md]: @benhylau  thoughts?

Comment (makeworld-the-better-one) [reply docs/MODULES.md]: It was like this before of course, but there's no harm in discussing it now. I tend to agree, it's easily abused by putting it here. I would just link to the public sites as I have done, and whether @benhylau wants to put the deprecated.systems stuff in the public repo is up to him.

**Comment by makeworld-the-better-one** [comment]
I have removed the deprecated.systems peering info in the CJDNS, Yggdrasil and SSB sections. I would suggest that the dep.sys pub is added to public peering though in SSB (and CJDNS?), as it is the only public pub that has mesh, that I know of at least.

**Comment by darkdrgn2k** [comment]
@makeworld-the-better-one 
https://github.com/tomeshnet/prototype-cjdns-pi/pull/356/files#diff-86c4f95673d4319d89af160b557e42b1R248

One last place it exists other then that it looks good

**Comment by darkdrgn2k** [comment]
Looks good lets merge!

thanks @makeworld-the-better-one

**Pull request merged**